### PR TITLE
net-misc/gns3-{gui,server}: version bump, cleanup

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

simple version bump for both gns3-gui and -server.

### CODE REVIEW

File: net-misc/gns3-gui/gns3-gui-2.2.36.ebuild [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..10} )
+PYTHON_COMPAT=( python3_{10..11} )
```
Comment (ceamac) [inline net-misc/gns3-gui/gns3-gui-2.2.36.ebuild:6]: Did upstream remove python 3.9 support?  For now only python 3.8 is masked and about to be removed, python 3.9 is still in tree and supported

**Comment by ceamac** [comment]
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/lib/python3.10/site-packages/gns3server/compute/docker/resources/bin/busybox
 * QA Notice: Pre-stripped files found:
 * /usr/lib/python3.10/site-packages/gns3server/compute/docker/resources/bin/busybox
```

**Comment by ceamac** [comment]
```
DistutilsNonPEP517Build: version 2.2.36: uses deprecated non-PEP517 build mode, please switch to DISTUTILS_USE_PEP517=...
```

**Review by ceamac — CHANGES_REQUESTED** [review]
Please fix the QA warning about busybox and the pkgcheck warnings about switching to 
DISTUTILS_USE_PEP517.

Other than that, both server and gui build fine and pass the tests for both python 3.10 and python 3.11.

Meantime upstream released 2.2.37, you can bump it further if you want.

**Comment by ceamac** [comment]
And since tests pass, I think you can also add a tag to close https://bugs.gentoo.org/876709

**Comment by mm1ke** [comment]
@ceamac 
Thanks for the review (and sorry for the late reply). I've updated my PR, everything should be fine now.
Regarding pyhton 3.9. I've removed it because on gentoo it's considered deprecated (was looking here: https://wiki.gentoo.org/wiki/Project:Python/Implementations) I could add it again (upstream didn't remove support), but i think python 3.10-3.11 should be fine?
Also added the Bug to the commit, actually forgot that before.

**Review by ceamac — APPROVED** [review]
Still need to do the DISTUTILS_USE_PEP517=setuptools for gns3-gui
```
net-misc/gns3-gui
  DistutilsNonPEP517Build: version 2.2.37: uses deprecated non-PEP517 build mode, please switch to DISTUTILS_USE_PEP517=...
```

But other than that looks good, so let's leave it for next time.

Thank you!

**Pull request closed without merging**